app/models/institution.rb

```diff
@@ -1,18 +1,9 @@
 class Institution < ApplicationRecord
-  self.table_name = "ucas_institution"
-  self.primary_key = "inst_code"
+  self.table_name = "institution"
 
   has_and_belongs_to_many :organisations,
-    join_table: :mc_organisation_institution,
-    association_foreign_key: :org_id,
-    foreign_key: :institution_code
+    join_table: :organisation_institution
 
-  has_many :ucas_courses,
-    foreign_key: :inst_code
-
-  has_many :institution_enrichments,
-    foreign_key: :inst_code
-
-  has_many :course_enrichments,
+  has_many :courses,
     foreign_key: :inst_code
```

Comment:
the FK of course on institution is institution_id (mapping to institution.id

Reply:
It seems like it's a dead association - i'll remove it.

Reply:
Fixed in 1c7edae